### PR TITLE
modal for transactions and categories

### PR DESCRIPTION
**Pull request by ruofanwei**
截圖 2023-11-26 下午8 36 40

截圖 2023-11-26 下午8 32 25

截圖 2023-11-26 下午8 33 12

### CODE REVIEW

File: backend/categories/migrations/__init__.py
Comment (KalleChen) [inline backend/categories/migrations/__init__.py:1]: Initial migration file is required

**Review by KalleChen — COMMENTED** [review]
you will need to put your app schema into the schema of project level
which is `backend/schema.py`

File: backend/categories/migrations/__init__.py
Comment (ruofanwei) [reply backend/categories/migrations/__init__.py:1]: fixed at [ac09b75](https://github.com/KernelKrakens/FinBuddy/pull/7/commits/ac09b7589c97ad60ef98e5a54d02740a21b9abe8), thanks

File: backend/categories/models.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Category(models.Model):
+    category_type = models.CharField(max_length=250)
```
Comment (KalleChen) [inline backend/categories/models.py]: 對 type 額外設一個選項？

Comment (ruofanwei) [reply backend/categories/models.py]: fix at [2979923](https://github.com/KernelKrakens/FinBuddy/pull/7/commits/297992397e1ca4093de737a2b2de8153c0d1272d), thanks

File: backend/backend/settings.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
-config = Config(".env")
+# config = Config(".env")
```
Comment (KalleChen) [inline backend/backend/settings.py:24]: Why this be changed?

Comment (ruofanwei) [reply backend/backend/settings.py:24]: 印象中是不需要用到這個

Comment (KalleChen) [reply backend/backend/settings.py:24]: 👍

**Review by KalleChen — APPROVED** [review]

**Pull request merged**